server/androidtv_server.py

```diff
@@ -0,0 +1,753 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import random
+import webapp2
+import time
+from google.appengine.ext import ndb
+
+
+# category data base model
+class Category(ndb.Model):
+    category = ndb.StringProperty(indexed=True, required=True)
+
+
+# Movie model in detailed view page
+class MovieClip(ndb.Model):
+    id = ndb.IntegerProperty(indexed=True, required=True)
+    category = ndb.StringProperty(indexed=True, required=True)
+    description = ndb.StringProperty(indexed=True, required=True)
+    source = ndb.StringProperty(indexed=False, required=True)
+    card = ndb.StringProperty(indexed=False, required=True)
+    background = ndb.StringProperty(indexed=False, required=True)
+    title = ndb.StringProperty(indexed=False, required=True)
+    studio = ndb.StringProperty(indexed=False, required=True)
+    rented = ndb.BooleanProperty(indexed=False, required=True)
+
+
+# movie model in overview page
+class MovieOverview(ndb.Model):
+    id = ndb.IntegerProperty(indexed=True, required=True)
+    category = ndb.StringProperty(indexed=True, required=True)
+    source = ndb.StringProperty(indexed=False, required=True)
+    card = ndb.StringProperty(indexed=False, required=True)
+    background = ndb.StringProperty(indexed=False, required=True)
+    title = ndb.StringProperty(indexed=False, required=True)
+    studio = ndb.StringProperty(indexed=False, required=True)
+
+
+# global variable to track database status
+# when the database is created, this flag will be toggled to true
+DATABASE_CREATED = False
```

Comment:
Does this global variable get lost when app server is restarted?

Reply:
Yes, you are absolutely right. So I removed this global variable, another global variable VIDEO_ID will be updated every time through an explicit get request to url/create